### DISCUSSION TITLE
MIMIC_IV creatinine_baseline MDRD est (units)

### DISCUSSION THREAD

**Discussion by aoifeboyle3004** [MIT-LCP/mimic-code#1704] [MIMIC-IV]
I am trying to group a patient cohort based on their eGFR on admission (group by mild/mod/severe renal imp) 
Have reviewed the extremely useful creatinine_baseline table. All of the MDRD output in this does not reflect true eGFR values? Lot are in the 0.7 range - have tried to convert to umol/L but that is not correct either (have 0 patients with an eGFR score of <66ml/min)?

Any idea what the unit is for the MDRD_est - it is not in true eGFR?

**Comment by alistairewj** [comment]
Hmm odd, perhaps a mistake in the code? The MDRD_est column is meant to use the simple formula: eGFR = 186 × Scr^(-1.154) × Age^(-0.203) × 0.742(Female)

calculated here: https://github.com/MIT-LCP/mimic-code/blob/main/mimic-iv/concepts/measurement/creatinine_baseline.sql
https://github.com/MIT-LCP/mimic-code/blob/8cb60286a514b30d56aa46a76fff3939689a3db4/mimic-iv/concepts/measurement/creatinine_baseline.sql#L17-L22

**Reply by alistairewj** [reply]
Oh, right, this is an estimate of the baseline creatining *using* the MDRD formula in reverse. So.. this is just a poorly named column!

**Reply by aoifeboyle3004** [reply]
That makes far more sense!! Thank you - now to figure this one out :)
